homeassistant/components/whirlpool/const.py

```diff
@@ -1,10 +1,63 @@
-"""Constants for the Whirlpool Sixth Sense integration."""
+"""Constants for the Whirlpool Appliances integration."""
 
 from whirlpool.backendselector import Region
+from whirlpool.washerdryer import MachineState, WasherDryer
 
 DOMAIN = "whirlpool"
 
 CONF_REGIONS_MAP = {
     "EU": Region.EU,
     "US": Region.US,
 }
+
+TANK_FILL = {
```

Comment:
This is an optional style suggestion, but I'd say for constants only used in a single platform just leave them in the platform